Products/ZenUI3/browser/resources/js/zenoss/template.js

```diff
@@ -98,23 +98,21 @@ beforeselectHandler = function(sm, node) {
 };
 
 updateDataSources = function(uid) {
```

Comment:
minor code optimization. Using "Zenoss.getCmp instead of Ext.getCmp to get components by "itemId".